tests/agents/test_static_analysis_enricher_mixin.py

```
@@ -17,17 +28,28 @@
+from static_analyzer.config import Language, NodeType
 from static_analyzer.node import Node
 
 
 def component(component_id: str, name: str) -> Component:
     return Component(name=name, description=name, key_entities=[], component_id=component_id)
 
 
-def enricher(repo_dir: Path = Path("/repo")) -> StaticAnalysisEnricherMixin:
+def enricher(
+    repo_dir: Path = Path("/repo"),
+    component_ownership: ComponentOwnershipIndex = ComponentOwnershipIndex({}),
```

Ruff (0.16.2)

[warning] Do not perform function call `ComponentOwnershipIndex` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)